backend/generalhw.pm

```diff
@@ -174,9 +174,11 @@ sub start_serial_grab ($self) {
 }
 
 sub stop_serial_grab ($self, @) {
-    return unless $self->{serialpid};
-    kill("-TERM", $self->{serialpid});
-    return waitpid($self->{serialpid}, 0);
+    return 0 unless $self->{serialpid};
+    eval { kill('-TERM', $self->{serialpid}) };
```

Comment:
```suggestion
    eval { kill '-TERM => $self->{serialpid} };
```

Reply:
I'd removed the braces but since the quoting around the signal cannot be avoided anyways I'd rather stay away from `=>` for this function.

Reply:
The quoting can be avoided